### ISSUE TITLE
Type permissions exists in middleware but is missing in Schema.

### ISSUE THREAD

**Issue by kamarajuPrathi** [maticzav/graphql-middleware#250]
When using grapql middleware with graphql shield permissions

**Comment by ax-vasquez** [comment]
We hit this issue earlier as well, however we found out the problem. 

Right now, we are early in development for a pretty large microservices application backend. We basically have a bunch of ApolloServers stitched together as our backend - since we are still pretty early in development, we haven't applied our shield permissions.

It turns out, if you call `applyMiddleware(schema, permissions)` when your permissions object has no rules in it, **things start misbehaving**. I'm sure this is not intended as it doesn't happen for me, but it **did** happen for my girlfriend (even though we both have the same OS and similar setup).

#### TL;DR

Don't pass "rule-free" permissions (e.g., ones that look like the one below) to the `applyMiddleware` call.
```
export const permissions = shield({
  Query: {},
  Mutation: {},
})
``` 
- **Big caveat:** _I am sure this is not the only way to encounter this error, but hopefully it at least sheds some light on the issue_

Until we are ready to start writing our rules, we are passing `null` as our permissions object and added some logic to bypass the call to `applyMiddleware(schema, permissions)` entirely.

**Comment by maticzav** [comment]
@ax-vasquez I think this should be a consequence of version mismatch. Could you try updating to the latest version?

**Comment by maticzav** [comment]
I am closing the issue as it seems inactive. Let me know if you still experience the problem and reopen the issue.
